README.md

```diff
@@ -388,6 +388,22 @@ Variable | Description | Example
 
 For setting the Google Cloud Credentials please read https://cloud.google.com/text-to-speech/docs/quickstart-protocol section "Before you begin" paragraph 1 to 5.
 
+### Access and Application logging
+
+For privacy reasons it is important to have the possibility to disable all logs about the users. To achieve this you can configure the log level for each component with these options:
+
+Variable | Description | Default value
+--- | --- | ---
+`PROSODY_LOG_LEVEL` | Controls which logs are output from prosody and associated modules. [Supported levels](https://prosody.im/doc/logging) | info
+`JVB_LOG_LEVEL` | Controls which logs are written to stdout. [Supported java log levels](https://docs.oracle.com/javase/8/docs/api/java/util/logging/Level.html) | INFO
+`JICOFO_LOG_LEVEL` | Controls which logs are written to stdout. [Supported java log levels](https://docs.oracle.com/javase/8/docs/api/java/util/logging/Level.html) | INFO
+`JIGASI_LOG_LEVEL` | Controls which logs are written to stdout. [Supported java log levels](https://docs.oracle.com/javase/8/docs/api/java/util/logging/Level.html) | INFO
+`JIBRI_LOG_LEVEL` | Controls which logs are written to stdout. [Supported java log levels](https://docs.oracle.com/javase/8/docs/api/java/util/logging/Level.html) | INFO
+`WEB_LOG_LEVEL` | Controls nginx logs written to stdout. INFO enables access logs, ERROR the error logs and OFF disables all (but [`emerg` level](https://nginx.org/en/docs/ngx_core_module.html#error_log) for error logs) | INFO
+`ETHERPAD_DISABLE_IP_LOGGING` | Disable IP logging | false
+`ETHERPAD_LOG_LEVEL` | Controls which logs are written to stdout. Supported levels: DEBUG, INFO, WARN, ERROR | INFO
```

Comment:
These change need to be applied to https://github.com/jitsi/handbook/blob/master/docs/devops-guide/docker.md (cf: #601)

Reply:
Oh nice, I will add this to the handbook